backend/app/assets/stylesheets/spree/backend/shared/_helpers.scss

```diff
@@ -0,0 +1,35 @@
+.block-table {
```

Comment:
`helpers` is a bit of an overloaded term. How about `utilities` or `utils`?

If you believe helpers is the most descriptive, I'm also fine with that.